Formula/couchdb.rb

```diff
@@ -78,37 +123,74 @@ def post_install
     end
   end
 
-  def caveats; <<-EOS.undent
+  def caveats
+    s = <<-EOS.undent
     To test CouchDB run:
         curl http://127.0.0.1:5984/
 
     The reply should look like:
         {"couchdb":"Welcome","uuid":"....","version":"#{version}","vendor":{"version":"#{version}-1","name":"Homebrew"}}
     EOS
+    s += "\n#{geocouch_caveats}" if build.with? "geocouch"
+    s
   end
 
-  plist_options :manual => "couchdb"
+  def geocouch_caveats; <<-EOS.undent
+    You can access, test, and configure your new geocouch with:
+
+      http://127.0.0.1:5984
+      http://127.0.0.1:5984/_utils/couch_tests.html?script/couch_tests.js
+      http://127.0.0.1:5984/_utils
+
+    To uninstall geocouch from your couchdb installation, uninstall couchdb
+    and re-install it without the '--with-geocouch' option.
+
+      brew uninstall couchdb
+      brew install couchdb
 
-  def plist; <<-EOS.undent
-    <?xml version="1.0" encoding="UTF-8"?>
-    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-    <plist version="1.0">
-    <dict>
-      <key>KeepAlive</key>
-      <true/>
-      <key>Label</key>
-      <string>#{plist_name}</string>
-      <key>ProgramArguments</key>
-      <array>
-        <string>#{opt_bin}/couchdb</string>
-      </array>
-      <key>RunAtLoad</key>
-      <true/>
-    </dict>
-    </plist>
+    To see these instructions again, just run 'brew info couchdb'.
     EOS
   end
 
+  plist_options :manual => "couchdb"
+
+  def plist
+    linked_geocouch_share = (HOMEBREW_PREFIX/"share/couchdb-geocouch")
+    if build.with? "geocouch"
+      #geocouch_plist = geocouch_share/"geocouch.plist"
+      s = File.read(pkgshare/"etc/launchd/org.apache.couchdb.plist.tpl.in")
+      s.gsub! "<string>org.apache.couchdb</string>", \
```

Comment:
The file is going to have to be read in anyway, since it's the contents which are returned by `plist`, not the location. And a copy would have to be made, since `pkgshare` should contain the original, unmodified file from the distribution. So it seems like using `inreplace` would make this munging longer and more verbose, since it would add a copy operation and add the file name as an argument to each replacement method call.